corehq/apps/app_manager/views/forms.py

```diff
@@ -169,7 +169,8 @@ def edit_form_actions(request, domain, app_id, module_id, form_id):
     if actions_use_usercase(form.actions) and not is_usercase_in_use(domain):
         enable_usercase(domain)
     response_json = {}
-    app.save(response_json)
+    if not actions_use_usercase(form.actions) or domain_has_privilege(request.domain, privileges.USER_CASE):
+        app.save(response_json)
```

Comment:
My mistake. I thought that when changes to the case management form were submitted, they only included normal actions (because [case-config-ui-2.js#L53-L56](https://github.com/dimagi/commcare-hq/blob/usercase_downgrade/corehq/apps/app_manager/static/app_manager/js/case-config-ui-2.js#L53-L56)) and not usercase actions (because [case-config-ui-2.js#L847-L847](https://github.com/dimagi/commcare-hq/blob/usercase_downgrade/corehq/apps/app_manager/static/app_manager/js/case-config-ui-2.js#L847-L847)). But you are right; it includes both.

So either I need to take this out, and just rely on the UI to make it difficult for users to save user case changes, or I need to parse the actions, and compare with current user case management to see if the user has modified anything. I guess it depends on how important it is to us that it is impossible for the user to make user case changes. I'll ping @amsagoff but I suspect the UI changes might be enough.